src/Raven.Server/NotificationCenter/Indexing.cs

```diff
@@ -26,6 +26,8 @@ public sealed class Indexing : IDisposable
 
         private MismatchedReferencesLoadWarning _mismatchedReferencesLoadWarning;
 
+        private readonly ConcurrentQueue<(string indexName, string fieldName)> _warningComplexFieldAutoIndexing = new();
```

Comment:
Maybe let's aggregate that into single alert? We could have a list of indexes and list of problematic fields there. We already do similar thing for other alerts - `MismatchedReferencesLoadWarning` is one example I think

Reply:
![image](https://github.com/ravendb/ravendb/assets/86351904/3b377f1e-43ae-4f80-8ae8-d38bdd4f538e)

Reply:
We are inflicting this on the user directly, no?

Reply:
Yes, as an indexing alert.